ubuntu/debian/control

```diff
@@ -41,6 +42,7 @@ Architecture: any
 Section: libdevel
 Depends: libgz-cmake3-dev,
        libgz-common5-core-dev,
+       libgz-common5-testing-dev,
```

Comment:
I don't think we need it here since it's only used for testing.

Reply:
Removed in https://github.com/gazebo-release/gz-fuel-tools8-release/pull/14/commits/e92310137973e7079612319433209e18badffa0a